### ISSUE TITLE
Videos from BBB serveur use MP4 and not webm

### ISSUE THREAD

**Issue by Lutice** [trahay/bbb-downloader#10]
Hello

If the video stored on bbb serveur use mp4 and not webm nothing is downloaded and there is an error.
To solve this problem you just need to replace all the webm by mp4 call in the bbb.py file
same thing to do in capture-full-replay.sh file

Probably it could be intersting to add a parameter to select webm/mp4 from command line
Regards
Fabrice

**Comment by olberger** [comment]
Or we could try and autodetect the format by looking at the metadata, or by checking the DOM content of the replay page, or in last resort, attempting and retrying in case of 404 response codes maybe...
We may to have the time to work on fixing that, but patches welcome, if anyone can.

**Comment by olberger** [comment]
AFAIU, the metadata doesn't allow detecting... and some installations may have both formats available for download.

I've added a new option, and the possibility for fallback to other format in case it doesn't work.

@geezot I guess https://github.com/trahay/bbb-downloader/commit/f9c55daac3491b46d8a7d993fd1f37ba7c5ac198 addresses your need. Closing then. Feel free to report if I made a mistake (having no way to test for your installation ;)
